content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -198,6 +198,73 @@ kubectl apply -f my-private-reg-pod.yaml
 kubectl get pod private-reg
 ```
 
+## Differences between Kubernetes and Docker
```

Comment:
I'd put all this detail in the concept page, and have the task page link to that page fragment.